libmisc/btrfs.c

```diff
@@ -0,0 +1,94 @@
+#include <linux/btrfs_tree.h>
+#include <linux/magic.h>
+#include <sys/statfs.h>
+
+#include "prototypes.h"
+
+
+static int run_btrfs_subvolume_cmd(const char *subcmd, const char *arg1, const char *arg2)
+{
+	int status = 0;
+	const char *cmd = "/sbin/btrfs";
+	const char *argv[] = {
+		strrchr(cmd, '/'),
```

Comment:
That feels like it shouldn't be done unconditionally. That seems like a good candidate for later bugs. :)

Reply:
This is not unconditional. The condition is on line 21 for "just in case" scenario. Maybe a little bit "fancy" and the condition should be moved up. It should need another variable then.

Let me know if you'd like me to refactor this or it's ok.